SwedbankPaySDK/Classes/SwedbankPayWebViewController.swift

```diff
@@ -169,21 +169,21 @@ extension SwedbankPayWebViewController : WKNavigationDelegate {
         }
     }
     
-    private func decidePolicyFor(url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+    private func decidePolicyFor(navigationAction: WKNavigationAction, url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
         attemptOpenUniversalLink(url) { opened in
             if opened {
                 decisionHandler(.cancel)
             } else {
-                self.decidePolicyForNormalLink(url: url, decisionHandler: decisionHandler)
+                self.decidePolicyForNormalLink(navigationAction: navigationAction, url: url, decisionHandler: decisionHandler)
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 126 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)